client/components/stream/related-video-stream.js

```
@@ -30,7 +30,13 @@ var exec = function () {
 				if (response.ok) {
 					return response.json();
 				} else {
-					throw new Error(`Brightcove proxy responded with ${response.status}`);
+
+					// Only throw an error if it's not a 4xx response
+					if (response.status.toString()[0] !== "4") {
```

Thanks Matt. I'm trying to understand this better ... 
(1) In this file, the `catch` completely ignores the error, so isn't it pointless to throw the error at all? 
(2) In the other file (related.js), we still want to pay attention to non-4xx errors, yeah?